### DISCUSSION TITLE
show transactions descending

### DISCUSSION THREAD

**Discussion by noaheyer** [firefly-iii/firefly-iii#7039] [General]
Hello Together

Please excuse me if I am wrong about this. This is the very first time I post something on GitHub. 

I wanted to ask if it would somehow be possible for me to see what are the biggest expenses of mine in the current month. It would be cool to be able to see the list in descending order. I have now looked around for a very long time but could not find an option that does just that.

Does this already exist and did I just overlook it or does anyone have a tip for me?

Thanks already!

**Comment by JC5** [comment]
What about the default financial report which has this?

**Reply by noaheyer** [reply]
Doesn't that just show the expenses of the accounts? If I have two accounts, I only see the total of each account. But this is not my question, I want to have a list from the largest to the smallest transaction (expense).

**Reply by JC5** [reply]
That's true. What you're looking for isn't readily available.

**Reply by noaheyer** [reply]
ok, pity

**Reply by yeagerghost** [reply]
@noaheyer The category report has this if I understand what you are asking.  I myself wanted something like that to see where I spend the most money.  It is a bit awkward however because the list shows up at the bottom below all the graphs

[Image from Demo site](https://raw.githubusercontent.com/yeagerghost/imagestore/main/expense-by-category-demo.png)

[Report link from Demo site](https://demo.firefly-iii.org/reports/category/6,4,1,5,41,40,42,2,3/7,3,2,1,10,8,4,5,6,9/20230101/20230131)

Maybe in a future release I am hoping we can have it kind of separate because I have quite a few categories and scrolling to the bottom is the only drawback

**Reply by yeagerghost** [reply]
@noaheyer Actually just looking around some more I realize that the image above which shows top 10 expenses is by DAY and not report time period.  That is to say, which single day in that time period had the most spent. For the cumulative expense by a category, if you want that, that is actually at the top.  Maybe you can try this report and see if it works for you 

[Cumulative total by category](https://raw.githubusercontent.com/yeagerghost/imagestore/main/categories-and-weird-characters.png)

[Same demo report link from above](https://demo.firefly-iii.org/reports/category/6,4,1,5,41,40,42,2,3/7,3,2,1,10,8,4,5,6,9/20230101/20230131)

@JC5 Maybe this was reported somewhere already but I thought I'd point it out since I am already in this thread with the report.  There are some weird characters that show up when I click to sort by spent

**Reply by JC5** [reply]
Those characters are part of the sorting in the table (0..9) and can sometimes overlap with the text in the header.

**Reply by yeagerghost** [reply]
OK thanks for letting me know
